### ISSUE TITLE
Returns the status code in the result

### ISSUE THREAD

**Issue by nitishm** [nitishm/vegeta-server#9]
The `/report` command doesn't return the `status_code` field which is available in the output from the `vegeta report` CLI command.

The problem is that the `status_code` block returns the HTTP status code (key) mapped to the code cound (value). Swagger OpenAPI 2.0 doesn't support dynamic keys.

One possible way is to extract the status code into a `map[string]string` type and then adapt it to the spec, where the spec could be of type 
```
      statusCodes:
        type: object
        properties:
          code:
            type: string
          count:
            type: integer
```

**Comment by andrysds** [comment]
hello! can i grab this issue?

i have been using go for a year now, but never use go-swagger
can u give me some hint for my start on solving the problem? i tried to explore the code but found no start point.

**Comment by nitishm** [comment]
@andrysds definitely! Let me know if you need any help.

**Comment by nitishm** [comment]
Fixed in #20
